File: buttons/buttons.py
def main():
    case_num = int(input())
    for _ in range(case_num):
        a, b, c = input().split()
        a = int(a)
        b = int(b)
        c = int(c)
        if a > b:
            print("First")
        elif b > a:
            print("Second")
        elif c % 2:
            print("First")
        else:
            print("Second")
    
    
if __name__ == "__main__":
    main()

File: buttons/buttons.c
#include <stdio.h>

int main()
{
    int case_num;
    scanf("%d", &case_num);
    for (int i = 0; i < case_num; i++)
    {
        int a, b, c;
        scanf("%d%d%d", &a, &b, &c);
        if (a > b)
        {
            printf("First\n");
        }

        else if (b > a)
        {
            printf("Second\n");
        }

        else if (c % 2)
        {
            printf("First\n");
        }

        else
        {
            printf("Second\n");
        }
    }
}
